### PR TITLE
iptables: wait for table lock

### PR DESCRIPTION
**Pull request by Mic92**

### CODE REVIEW

File: taps/tapManager.go [hunk 1]
```diff
@@ -124,7 +124,7 @@ func ConfigIPtables(tapName, hostIface string) error {
 		}
 	}
 	cmd := exec.Command(
-		"sudo", "iptables", "-t", "nat", "-A", "POSTROUTING", "-o", hostIface, "-j", "MASQUERADE",
+		"sudo", "iptables", "--wait", "-t", "nat", "-A", "POSTROUTING", "-o", hostIface, "-j", "MASQUERADE",
```
Comment (Mic92) [inline taps/tapManager.go:127]: I have only seen this so far for MASQUERADE. Not sure if the other needs it as well.

Comment (Mic92) [reply taps/tapManager.go:127]: I see also for the other commands now.

**Comment by ustiugov** [comment]
@Mic92 thanks for the contribution! 

can you explain the issue a bit in the PR description? did you notice any failure in your setting?

also, according to the manual, iptables can take a max waiting-time argument. I think it would make sense to have a 1-min timeut to prevent threads to hang. what do you think?

**Comment by ustiugov** [comment]
also, please sign the commit.

**Comment by Mic92** [comment]
done.

**Review by ustiugov — DISMISSED** [review]
looks good to me. @Mic92 please rebase and let's merge the changes

**Comment by Mic92** [comment]
done

**Comment by ustiugov** [comment]
thank you for your contribution! I will merge it as soon as the tests pass

**Review by ustiugov — APPROVED** [review]

**Pull request merged**